bin/ch-test

```diff
@@ -516,6 +521,8 @@ vset CH_TEST_IMGDIR   "$imgdir"        "$CH_TEST_IMGDIR"   /var/tmp/img \
                       21 'unpacked images dir'
 vset CH_TEST_TARDIR   "$tardir"        "$CH_TEST_TARDIR"   /var/tmp/tar \
                       21 'packed images dir'
+vset CH_ARCHIVE_FMT   "$archive_fmt"   "$CH_ARCHIVE_FMT"   squashfs \
```

Comment:
Call it `CH_TEST_PACK_FMT` or `CH_PACK_FMT`? Do you think the variable should be used only for testing, or generally, keeping in mind we'll be moving to your new user-friendly scripts soon?

Reply:
The default will need to be what is installed, not simply `squashfs`.